package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-debug",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Don't bump version on PRs. It's up to the maintainer to choose when and what version to use.